spec/views/civil_servants/services/show.html.slim_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:org_address) { organization.address }
   let!(:expense_sheet) do
     create(:expense_sheet, :locked,
-           service: service, beginning: service.beginning, ending: service.beginning.at_end_of_month)
+           service: service, beginning: service.beginning, ending: service.ending)
```

Comment:
Y dis?

Reply:
Because there was a wierd error. The spec failed cause the expense sheet was not during the service duration